src/disco/FixedWidthField/Integer/src/parseInteger.hpp

```diff
@@ -32,7 +32,7 @@ parseInteger( Iterator& it, uint16_t& position, Signed ){
 
   const auto magnitude = parseInteger( it, position, Unsigned() );
 
-  if ( unlikely( magnitude > std::numeric_limits< int64_t >::max() ) ){
+  if ( unlikely( magnitude > (uint64_t) std::numeric_limits< int64_t >::max() ) ){
```

Comment:
Why can't we just do:
```suggestion
  if ( unlikely( magnitude > std::numeric_limits< uint64_t >::max() ) ){
```
When I did this, everything compiled just fine with gcc 9.2 and LLVM (don't know the version.

Reply:
I don't know why this got posted twice.